src/ansys/dpf/gate/data_sources_grpcapi.py

```diff
@@ -60,6 +60,18 @@ def data_sources_set_domain_result_file_path_utf8(dataSources, name, id):
         request.data_sources.CopyFrom(dataSources._internal_obj)
         _get_stub(dataSources._server).Update(request)
 
+    @staticmethod
+    def data_sources_set_domain_result_file_path_with_key_utf8(dataSources, name, key, id):
+        from ansys.grpc.dpf import data_sources_pb2
```

Comment:
@PProfizi Shouldn't this be done in the place where this code is maintained?

Reply:
@rafacanton yes, it will. This is to test the change. we can also merge it from here while waiting for it to come from the server-sync. It will be crushed anyway.

Reply:
I agree with @rafacanton, we should even gitignore this files because if you're not there to check that manually, then we can have problems: when updating dpf-standalone, these files will be erased and and we will have failing unit tests

Reply:
@rafacanton @cbellot000 I propose that the `_grpcapi.py` files of `ansys/dpf/gate/` should be versioned in `pydpf-core` directly. Then all the server-sync does is provide a `ansys/dpf/gate/generated` folder to replace the ones in `pydpf-core`.
If we gitignore these files, then they will be ignored and not tracked.